src/containers/movie/AddMovieOMDB.js

```diff
@@ -0,0 +1,68 @@
+
+  
+import React, {Fragment} from 'react';
+import {connect} from 'react-redux';
+import TextField from '@material-ui/core/TextField';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import Button from '@material-ui/core/Button';
+import axios from 'axios';
+import { getAllGenres, postMovieOmdb, searchMovieOmdb } from '../../store/actions/MovieActions';
+
+class AddMovieOMDB extends React.Component{
+    state = {
+        searchTitle: "",
+        searchYear: ""
+    }
+    
+    handleChange(event, field){
+        this.setState({[field]: event.target.value});
+    }
+    handleSearchMovie = () =>{
+        this.props.searchMovieOmdb({title: this.state.searchTitle, year: this.state.searchYear});
+    }   
+    handlePostMovie = ()=>{
+        this.props.postMovieOmdb(this.props.searchResult);
+    }
+    render(){
+        return (
+        <div style={{marginLeft:'5%'}}>
+            <h2>OMDB movie search</h2>
+            <TextField style={{marginBottom:'5%', marginRight:'5%'}} value={this.state.searchTitle} onChange={(event) => this.handleChange(event, "searchTitle")} id="standard-basic" label="Title" required/>
+            <TextField style={{marginBottom:'5%'}} value={this.state.searchYear} onChange={(event) => this.handleChange(event, "searchYear")} id="standard-basic" label="Year" required/>
+            <Button onClick={this.handleSearchMovie} variant="contained" color="primary">
+                Search
+            </Button>
+
+            <br></br>
+            <TextField  value={this.props.searchResult.title} id="standard-basic" label="Title" />
+            <p style={{color:'red'}} >{this.props.errors.title && this.props.errors.title}</p>
```

Comment:
radi ovako `{this.props.errors.title && <p style={{color:'red'}} > this.props.errors.title</p>}`

Reply:
nema potrebe da imas element koji je prazan